### PR TITLE
Cloud changelog - Aug 25th

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
-- [Release 2022-07-13](#release-2022-07-13)
+
+## Release 2022-08-25
+
+### Highlights
+
+### Improvements
+
+#### User Interface (UI)
+ - Added the "Save" option to the post menu.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:21]:
```suggestion
 - Added the **Save** option to the post menu.
```

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+#### Administration
+ - Started tracking the join time of team members and added a new API endpoint to retrieve information about team members who have joined during a given time.
+ - Introduced an optional ``shouldRender`` function parameter to ``registerchannelHeaderMenuAction`` plugin function. This allows menu items to conditionally render depending on the current state prior to rendering.
+
+### Bug Fixes
+ - Fixed an issue where exports generated via mmctl without attachments still included the file properties in the post, so they couldn't be imported.
+ - Fixed an issue that caused a crash when unread posts were fetched.
+
+### Known Issues
+ - Mentions incorrectly shows users as not in a channel [MM-44157](https://mattermost.atlassian.net/browse/MM-44157).
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:34]:
```suggestion
 - Mentions incorrectly show users as not in a channel [MM-44157](https://mattermost.atlassian.net/browse/MM-44157).
```

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - System Roles shows **License** and **Environment** as possible permissions, but they are always hidden in Cloud.
+ - The Playbooks left-hand sidebar does not update when a user is added to a run or playbook without a refresh.
+ - The runs and playbooks in the Playbooks left-hand sidebar does not have dot-menus that allow interaction with each item [MM-44752](https://mattermost.atlassian.net/browse/MM-44752).
+ - On the new Boards RHS from the channel Apps Bar, channel members who are not admins of the board are incorrectly able to see the "unlink" board button. However, clicking on the button will not actually unlink the board unless the user is a board admin [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:38]:
```suggestion
 - On the new Boards RHS from the channel Apps Bar, channel members who are not board admins are incorrectly able to see the **Unlink** board button. However, selecting the button doesn't actually unlink the board unless the user is a board admin [issue-focalboard-3600](https://github.com/mattermost/focalboard/issues/3600).
```

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,39 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - On Boards, clicking on `+ New` button below a column on the Kanban view does not always create a new card. As a workaround, set a new default card template by going to the dropdown menu from the blue `New` button on the header of the board, then open the Options Menu on any card template and select "Set as default" [issue-focalboard-3676](https://github.com/mattermost/focalboard/issues/3676).
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:39]:
```suggestion
 - On Boards, selecting the **+ New** button below a column on the Kanban view doesn't always create a new card. As a workaround, set a new default card template by going to the dropdown menu from the blue **New** button on the header of the board, opening the **Options** menu on any card template, and selecting **Set as default** [issue-focalboard-3676](https://github.com/mattermost/focalboard/issues/3676).
```

**Review by cwarnermm — COMMENTED** [review]
Feedback provided inline

**Review by spirosoik — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Pull request merged**